packages/graphql-contentful-helpers/src/createContext.ts

```diff
@@ -75,7 +119,7 @@ const createContext = async ({ config }: CreateContextProps): Promise<ApolloCont
     : null;
 
   return {
-    contentful,
+    contentful: clients,
```

Comment:
This should either be renamed to cms, or we should have a specific field for sanity, in addition to contentful. Since the clients have different APIs, specifically naming them differently is probably the best option